src/org/labkey/test/tests/SecurityTest.java

```diff
@@ -128,6 +129,7 @@ public void testSteps() throws IOException
         }
 
         clonePermissionsTest();
+        projectUsersTest();
```

Comment:
`projectUsersTest` should go into its own `@Test` method, if able. It doesn't appear to depend on or impact any of the other tests.

Reply:
When I pulled it out, it flopped because `clonePermissionsTest()` in `testSteps()` creates the project in the first place. In this case, is it alright to piggyback off of that created project, or should I be setting up my own project/factoring out the project creation into a setup fn?